Tests/Algorithm/Framework/Portfolio/ConfidenceWeightedPortfolioConstructionModelTests.cs

```diff
@@ -401,6 +410,36 @@ private void SetPortfolioConstruction(Language language, QCAlgorithm algorithm)
             algorithm.PortfolioConstruction.OnSecuritiesChanged(_algorithm, changes);
         }
 
+        private void SetPortfolioConstruction(Language language, QCAlgorithm algorithm, TimeSpan span)
+        {
+            algorithm.SetPortfolioConstruction(new ConfidenceWeightedPortfolioConstructionModel(span));
+            if (language == Language.Python)
+            {
+                using (Py.GIL())
+                {
+                    var name = nameof(ConfidenceWeightedPortfolioConstructionModel);
+                    var instance = Py.Import(name).GetAttr(name).Invoke(span.ToPython());
+                    var model = new PortfolioConstructionModelPythonWrapper(instance);
+                    algorithm.SetPortfolioConstruction(model);
+                }
+            }
+        }
+
+        private void SetPortfolioConstruction(Language language, QCAlgorithm algorithm, Func<DateTime, DateTime> func)
```

Comment:
Could make the last argument `dynamic argument` and use `((object)argument).ToPython()` to avoid the method duplication